app/Http/Controllers/ClassController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
```

Comment:
make all uses below each other don't add empty lines between, this is a standard